frontend/src/metabase/components/BrowseApp.jsx

```diff
@@ -32,7 +32,14 @@ export class SchemaBrowser extends React.Component {
       <Box>
         <Schema.ListLoader query={{ dbId }}>
           {({ schemas }) =>
-            schemas.length > 1 ? (
+            schemas.length === 1 ? (
+              <TableBrowser
+                {...this.props}
+                params={{ ...this.props.params, schemaName: schemas[0].name }}
+                // hide the schema since there's only one
+                showSchemaInHeader={false}
+              />
+            ) : (
```

Comment:
In the `schemas.length === 0` case, will the empty `<Grid>` below display "correctly"/nicely?

Reply:
It matched the previous behavior of displaying nothing (under the breadcrumb header), but showing a note for empty dbs is definitely better! I just added one.